client/js/netflix/watchScreen.js

```diff
@@ -111,11 +103,13 @@ var mutationObserver = new MutationObserver(function(mutations) {
             const title = getTitle();
             
             // Get the cast details
-            // const movieDetails = getCastDetails(title);
-            const casts = getCastDetails1(title);
-            console.log("printing cast details ", casts);
-            console.log(typeof(casts));
-            addToWatchScreen(casts)
+            const response = getCastDetails(title).then(result => {
+                    var castString = result.cast;
```

Comment:
shall we check for null here?

let castString = result ? result.cast : null;

and use let or const if possible.

and then below you can do

const casts = castString && castString !== 'N/A' ? castString.split(',') : 'Cast Information not available';
addToWatchScreen(casts)

Reply:
Yeah makes sense! I will do that